src/controllers/userController.js

```diff
@@ -158,6 +158,78 @@ const listUsers = async (request, response) => {
   }
 };
 
+/*
+   * follow users feature
+   * @param {Object} req - server request
+   * @param {Object} res - server response
+   * @return {Object} - custom response
+   */
+const follow = async (req, res) => {
+  const { userId } = req.params;
+  const authUserId = req.user.id;
+  if (userId === authUserId) {
+    return responseMessage(res, 403, { error: 'You cannot follow yourself' });
+  }
+
+  try {
+    const isFollower = await findFollower(userId, authUserId);
```

Comment:
don't you think follower is a better variable name? `isFollower` kinda makes it seem like I'm expecting a boolean response

Reply:
Yeah, I agree.